src/test/java/org/sagebionetworks/bridge/exporter/dynamo/DynamoHelperTest.java

```diff
@@ -295,29 +299,44 @@ public void bootstrapStudyIdsToQueryTestNormal() {
 
         // mock request
         BridgeExporterRequest request;
-        List<String> studyIdsToUpdate;
+
+        Map<String, DateTime> studyIdsToUpdate;
         // daily
-        request = new BridgeExporterRequest.Builder().withDate(UPLOAD_DATE_OBJ).build();
+        request = new BridgeExporterRequest.Builder().withEndDateTime(UPLOAD_END_DATE_TIME_OBJ)
+                .withExportType(ExportType.DAILY).build();
         studyIdsToUpdate = dynamoHelper.bootstrapStudyIdsToQuery(request, UPLOAD_END_DATE_TIME_OBJ);
 
         assertEquals(studyIdsToUpdate.size(), 2);
-        assertEquals(studyIdsToUpdate.get(0), "ddb-foo");
-        assertEquals(studyIdsToUpdate.get(1), "ddb-bar");
+        assertEquals(studyIdsToUpdate.get("ddb-foo").getMillis(), UPLOAD_START_DATE_TIME_OBJ.getMillis());
+        assertEquals(studyIdsToUpdate.get("ddb-bar").getMillis(), UPLOAD_START_DATE_TIME_OBJ.getMillis());
```

Comment:
We should also compare dates and not just epoch milliseconds, so we can make sure we're setting timezone correctly. (Here and everywhere else in this test class.)